grafana/types.json

```diff
@@ -1858,6 +1858,16 @@
         "overrides": []
       }
    },
+   "bitsps_panel":{
+      "class":"iops_panel",
+      "fieldConfig": {
+        "defaults": {
+          "class": "fieldConfig_defaults",
+          "unit": "decbits"
```

Comment:
never heard of such a unit.
Apparently it's in Grafana (https://github.com/grafana/grafana/blob/cb3fc369fe238ce9763ab2825bacf81fcfba34ab/packages/grafana-data/src/valueFormats/categories.ts#L507 ) 

Oh well.